<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selector de Servidores</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
      border-image: url(:/images/fondo.jpg) 0 0 0 0 stretch stretch;
}
QPushButton {
    background-color: #2C1810;  
    border: 4px solid #FFB626;  
    border-style: solid;
    border-bottom-width: 8px;   
    color: #FFD700;            
    font-family: &quot;Press Start 2P&quot;, &quot;VT323&quot;, &quot;Consolas&quot;; 
    font-size: 28px;
    text-transform: uppercase;
    padding: 15px 30px;
    margin: 10px;
    border-radius: 0px;        
    font-weight: bold;
}
QPushButton:hover {
    background-color: #3C2820;  
    border-bottom-width: 10px;  
}
QPushButton:pressed {
    background-color: #1C0800;  
    border-bottom-width: 4px;   
}
QPushButton:disabled {
    background-color: #1A1A1A;
    border-color: #666666;
    color: #888888;
}
QLabel {
    background-color: #2C1810;  
    border: 4px solid #FFB626;  
    border-style: solid;
    border-bottom-width: 8px;   
    color: #FFD700;            
    font-family: &quot;Press Start 2P&quot;, &quot;VT323&quot;, &quot;Consolas&quot;; 
    font-size: 28px;
    text-transform: uppercase;
    padding: 15px 30px;
    margin: 10px;
    border-radius: 0px;        
}
QLineEdit, QComboBox {
      border: 4px solid #df9f1f; 
      color: #df9f1f; 
      font-family: &quot;Arial Black&quot;;
      font-size: 35px;
      font-weight: bold;
      text-transform: uppercase;
      padding: 15px 30px;
      margin: 10px;
}
QListView { 
      background-color: #1E1E1E; 
      color: #FFFFFF; 
      border: 2px solid #333333; 
      border-radius: 8px; 
      padding: 5px; 
      font-size: 16px; 
      font-weight: bold; 
}
QListView::item { 
      padding: 15px 20px; 
      margin: 4px; 
      border: none; 
      border-radius: 6px; 
      background-color: #2A2A2A; 
}
QListView::item:hover { 
      background-color: #383838; 
      border-left: 3px solid #666666; 
}
QListView::item:selected { 
      background-color: #3A3A3A; 
      border-left: 3px solid #FFFFFF; 
      color: #FFFFFF; 
}
QListView::item:selected:active { 
      background-color: #404040; 
}
QListView::item:alternate { 
      background-color: #262626; 
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="Main">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="Create">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="EditorButton">
            <property name="text">
             <string>Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Join">
            <property name="text">
             <string>Join</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Exit">
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CreateMenu">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Local Players</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="player1namecreate">
            <property name="text">
             <string>Player 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="LimitLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Player Limits</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="LimitPlayerQuantity">
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="CreateLocalPlayers">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="player2namecreate">
            <property name="text">
             <string>Player 2 </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LobbyName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Lobby Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="connectCreat">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BackButton">
          <property name="text">
           <string>Volver</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Servidores">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="serverList"/>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Volver">
          <property name="text">
           <string>Volver</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Connecting">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="LabelConnect">
          <property name="text">
           <string>Players</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="QuantityPlayers">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Player1NameJoin">
          <property name="text">
           <string>Player 1 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Player2NameJoin">
          <property name="text">
           <string>Player 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Connect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Lobby">
       <layout class="QVBoxLayout" name="lobbyLayout">
        <item>
         <widget class="QLabel" name="lobbyTitle">
          <property name="styleSheet">
           <string notr="true">QLabel {
      font-size: 24px;
      font-weight: bold;
      color: #df9f1f;
}</string>
          </property>
          <property name="text">
           <string>Lobby</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playerListLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
      font-size: 18px;
      color: #ffffff;
}</string>
          </property>
          <property name="text">
           <string>Jugadores Conectados:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="playerList">
          <property name="styleSheet">
           <string notr="true">QListView {
      background-color: #2E2E2E;
      color: #FFFFFF;
      border: 1px solid #444444;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startGameButton">
          <property name="text">
           <string>Iniciar Juego</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leaveLobbyButton">
          <property name="text">
           <string>Salir del Lobby</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
